src/ai/backend/test/templates/auth/keypair.py

```diff
@@ -1,16 +1,20 @@
 from contextlib import asynccontextmanager as actxmgr
-from typing import AsyncIterator, override
+from typing import AsyncIterator, Optional, override
 
 from ai.backend.client.config import APIConfig
 from ai.backend.client.session import AsyncSession
 from ai.backend.test.contexts.auth import EndpointContext, KeypairContext
 from ai.backend.test.contexts.client_session import ClientSessionContext
 from ai.backend.test.templates.template import (
+    TestTemplate,
     WrapperTestTemplate,
 )
 
 
 class KeypairAuthTemplate(WrapperTestTemplate):
+    def __init__(self, template: Optional[TestTemplate] = None, *args, **kwargs) -> None:
+        super().__init__(template, *args, **kwargs)
```

Comment:
I would like to avoid using args and kwargs unless it is an unavoidable situation.